src/Imdb/Calendar.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+#############################################################################
+# IMDBPHP                              (c) Giorgos Giagas & Itzchak Rehberg #
+# written by Giorgos Giagas                                                 #
+# extended & maintained by Itzchak Rehberg <izzysoft AT qumran DOT org>     #
+# http://www.izzysoft.de/                                                   #
+# ------------------------------------------------------------------------- #
+# IMDBPHP TOP CHARTS                                      (c) Ricardo Silva #
+# written by Ricardo Silva (banzap) <banzap@gmail.com>                      #
+# http://www.ricardosilva.pt.tl/                                            #
+# ------------------------------------------------------------------------- #
+# IMDBPHP UPCOMMING RELEASES (based on IMDBPHP TOP CHARTS (c)Ricardo Silva) #
+# written/modified/extended by Ed (github user: duck7000)                   #
+# ------------------------------------------------------------------------- #
+# This program is free software; you can redistribute and/or modify it      #
+# under the terms of the GNU General Public License (see doc/LICENSE)       #
+#############################################################################
+
+namespace Imdb;
+
+/**
+ * Obtaining information about upcomming movie releases as seen on imdB
+ * https://www.imdb.com/calendar
+ * @author Ricardo Silva (banzap) <banzap@gmail.com>
+ * @author Ed (github user: duck7000)
+ */
+class Calendar extends MdbBase
+{
+
+    protected $page = null;
+
+    /**
+     * Get upcomming movie releases as seen on imdB
+     * @return array (array[string date, array releases] of arrays[title,year,imdbid])
+     * @parameter $country, This defines wich country list is returned
+     * for example de, nl, us as they appear on https://www.imdb.com/calendar
```

Comment:
```suggestion
     * for example DE, NL, US as they appear on https://www.imdb.com/calendar
```

Reply:
Mm you are right about this, i changed it, but it also works with lowercase